<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthForm</class>
 <widget class="QWidget" name="AuthForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../Users/User/Documents/Tempовая хуйня/easter egg.jpg</normaloff>../../../../../Users/User/Documents/Tempовая хуйня/easter egg.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_4name">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="toolButtonback_to_auth">
         <property name="text">
          <string>  &lt;-  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_reg">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Регистрация</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ok">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QRadioButton" name="radioButton_student">
       <property name="text">
        <string>Студент</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QRadioButton" name="radioButton_teacher">
       <property name="text">
        <string>Учитель</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_1log">
       <property name="placeholderText">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_3email">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>email</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit_2pass">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="lineEdit_5surname">
       <property name="placeholderText">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QDial" name="dial"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_2pass</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>radioButton_teacher</tabstop>
  <tabstop>lineEdit_3email</tabstop>
  <tabstop>toolButtonback_to_auth</tabstop>
  <tabstop>pushButton_reg</tabstop>
  <tabstop>lineEdit_5surname</tabstop>
  <tabstop>lineEdit_4name</tabstop>
  <tabstop>lineEdit_1log</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
